templates/docs/examples/patterns/split-list.html

```diff
@@ -0,0 +1,522 @@
+            compliance for Ubuntu.
+          </p>
+        </div>
+      </div>
+
+      <div>
+        <div>
+          <hr>
+        </div>
+      </div>
+
+      <div>
+        <div>
+          <h5>
+            Sales enablement
+          </h5>
+        </div>
+
+        <div>
+          <p>
+            Canonical can help train your field sales and presales teams and
+            provide you with sales collateral and permission to use the Canonical
+            and Ubuntu trademarks in your own materials.
+          </p>
+        </div>
+      </div>
+    </div>
+  </div>
+</div>
+
+<h6>full width, full width, 25-25-75 (Jinja macro)</h6>
+
+{{ split_list(variant=1,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
+   }, {
+     'title': 'Sales enablement',
+     'description': 'Canonical can help train your field sales and presales teams and provide you with sales collateral and permission to use the Canonical and Ubuntu trademarks in your own materials.'
+   }]) }}
+
+<h6>full width, 50-50, 25-25-75 (Jinja macro)</h6>
+
+{{ split_list(variant=2,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
+   }, {
+     'title': 'Sales enablement',
+     'description': 'Canonical can help train your field sales and presales teams and provide you with sales collateral and permission to use the Canonical and Ubuntu trademarks in your own materials.'
+   }]) }}
+
+<h6>full width, 50-50, 50-50 (Jinja macro)</h6>
+
+{{ split_list(variant=3,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
+   }, {
+     'title': 'Sales enablement',
+     'description': 'Canonical can help train your field sales and presales teams and provide you with sales collateral and permission to use the Canonical and Ubuntu trademarks in your own materials.'
+   }]) }}
+
+<h6>full width, full width, 50-50 (Jinja macro)</h6>
+
+{{ split_list(variant=4,
+   title='H2 - this can take up to two lines, but best to stick to one line.',
+   list_items=[{
+     'title': 'Rich portfolio of products',
+     'description': 'Resell the full range of Canonical’s portfolio of private and public cloud products: Openstack, Kubernetes, IoT, support, security and compliance for Ubuntu.'
```

Comment:
We need a full HTML support here. This may be paragraphs, but also may be lists, images, etc (as per Lyubos latest designs).

Would it work for it to be multiline strings?

Can this content be defined elsewhere and referenced here (via some variables) potentially? (like when it will be taken from some kind of CMS or other external source).